frontend/src/data/utils/datasetutils.ts

```diff
@@ -221,5 +221,30 @@ export function maybeApplyRowReorder(rows: Row[], breakdowns: Breakdowns) {
       UNKNOWN_HL
     );
   }
+
+  if (breakdowns.hasOnlyAge()) {
+    finalRows.sort(sortAgeParsedNumerically);
+  }
+
   return finalRows;
 }
+
+export const sortAgeParsedNumerically = (l: any, r: any) => {
```

Comment:
Can you add comments to this function explaining what inputs it accepts for the ages?

Reply:
fixed